### ISSUE TITLE
Typo in the localization files

### ISSUE THREAD

**Issue by TheChilliPL** [Lattyware/massivedecks#168]
In the localization files, there are some small language mistakes I'll probably fix along with my Polish translation, but there's also a typo in a variable name:

```elm
        ViewConfgiuration -> [ Text "Configure" ]
--              ^^
        ViewConfgiurationDescription -> [ Text "Switch to view the game's configuration." ]
--              ^^
```

It appears in every language file in [this directory](../tree/master/client/src/elm/MassiveDecks/Strings/Languages/).

**Comment by Lattyware** [comment]
Yeah, this is an easy fix with a refactor, it isn't important as it isn't user-facing at all. I'll fix it with whatever the next commit I make is. Thanks for the heads-up.

**Comment by Lattyware** [comment]
Forgot to close this one, but this is fixed as of a3a62fba2e3718dfa1294fd621ecc4d9eb2b5e5e.
